src/components/views/ApplicationViews.js

```diff
@@ -80,7 +101,10 @@ export const ApplicationViews = () => {
         }
       />
 
-      <Route path="*" element={<Navigate to={"/"} state={{ from: location }} replace />} />
+      {/* <Route
+        path="*"
+        element={<Navigate to={"/"} state={{ from: location }} replace />}
+      /> */}
```

Comment:
This should not be commented out, as it allows users to access pages that do not exist.